### PR TITLE
20230327: regression analyses conducted

### PR DESCRIPTION
**Pull request by nick-hadacek**

### CODE REVIEW

File: projects/2023-NH/analysis.script_hadacek.R [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 
-### LOADING PACKAGES ###
+# LOADING PACKAGES ####
 
 install.packages("ggplot2") #for data visualization
 install.packages("dplyr") #for data management
 install.packages("cowplot") #combining graphs if needed
 install.packages("psych") #easier correlation
 install.packages("apaTables") #APA-ready tables
-install.packages("haven") #i don't remember what this was for
```
Comment (nguyenllpsych) [inline projects/2023-NH/analysis.script_hadacek.R:14]: {haven} is typically used for reading in .sav files and {readr} for .csv files, but you have {rio} for this already.

File: projects/2023-NH/analysis.script_hadacek.R [hunk 2]
```diff
@@ -38,74 +38,114 @@ sessionInfo()
...
+simple_regression <- lm(consci_dif ~ pid, data = jointdataset)
+summary(simple_regression)
+#interpreting
+  #consci_dif=(-.25410)pid+.68551
+  #p=1.523e-06
+  #For every increase in rigid perfectionism of 1, the difference between 
+  #peer-reported conscientiousness and self-reported conscientiousness decreases 
+  #by -0.2541. Given that rigid perfectionism ranges from 1-4 and 
+  #conscientiousness ranges from 1-5, this is a moderate-sized correlation 
+  #(r=-0.3189428). The result is also statistically significant (p<.05).
```
Comment (nguyenllpsych) [inline projects/2023-NH/analysis.script_hadacek.R:125]: ha! interesting

File: projects/2023-NH/analysis.script_hadacek.R [hunk 3]
```diff
@@ -38,74 +38,114 @@ sessionInfo()
...
+
+## multiple regression ####
+#multiple_regression <- lm(outcome_variable ~ pred1 + pred2 + pred.etc, data = data)
+#for moderator, use * instead of +
+
+multiple_regression <- lm(consci_dif ~ pid + pil + neuro + withdraw, 
+                          data = jointdataset)
+summary(multiple_regression)
+#interpreting
+  #consci_dif=(-.33045)pid+1.79231
```
Comment (nguyenllpsych) [inline projects/2023-NH/analysis.script_hadacek.R:145]: the formula for multiple regression actually includes all the other coefficients.

for example: let a be the coefficient (slope) for pid, b be the coefficient for pil, c for the coefficient for neuro, and d for withdraw -> the formula is:
consci_dif = a*pid + b*pil + c*neuro + d*withdraw + intercept

Comment (nick-hadacek) [reply projects/2023-NH/analysis.script_hadacek.R:145]: ohhhhhh ok i was a little confused about that but that makes so much sense, thank you!

**Pull request merged**